### PR TITLE
[REBRAND] Adds a new Pulsar Image to Readme

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR simply adds a new image to the Readme for this repo.

This readme is a bit outdated at this point, and while is still true to how the experience should feel on Pulsar, seemed appropriate that we update this for something that's truly ours. Especially since I've seen some newcomers dissuaded by us still having Atom's readme image.

I've taken the image on the most recent build of Pulsar, and the code is for the Pulsar backend. Additionally tried to, like the original, include the autocomplete features that are built into the editor.  If we don't think this image is suitable we can change it or close this PR.

### CODE REVIEW

**Comment by Sertonix** [comment]
Could you remove a few pixed at the left side of the image? There is a bit of a background image visible.

**Comment by confused-Techie** [comment]
> Could you remove a few pixed at the left side of the image? There is a bit of a background image visible. Edit: I think [flemshot](https://flameshot.org/) supports making a screenshot from a specific window

Alright I've removed the extra content on the side. How does it look?

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
 <!---------------------------{ Images }--------------------------->
 
-[Preview]: https://user-images.githubusercontent.com/378023/49132478-f4b77680-f31f-11e8-9e10-e8454d8d9b7e.png 'Preview of the editor.'
+[Preview]: https://github.com/pulsar-edit/pulsar/blob/master/resources/readme.png 'Preview of the editor.'
```
Comment (Sertonix) [inline README.md:67]: I reccommend using a relative path here
```suggestion
[Preview]: resources/readme.png 'Preview of the editor.'
```

Comment (confused-Techie) [reply README.md:67]: Yeah fair enough, I'll commit this one

**Review by Sertonix — APPROVED** [review]

**Review by DeeDeeG — APPROVED** [review]
Looks reasonable to me in preview: https://github.com/pulsar-edit/pulsar/blob/3d0ef897cb5080d91e64606c3b1dfbdef1cab5ca/README.md

**Pull request merged**